### PR TITLE
Run tools on PHP 8.2

### PR DESCRIPTION
**Pull request by derrabus**
Follows doctrine/.github#35

### CODE REVIEW

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ on:
 
 jobs:
   coding-standards:
-    uses: "doctrine/.github/.github/workflows/coding-standards.yml@2.1.0"
+    uses: "doctrine/.github/.github/workflows/coding-standards.yml@main"
```
Comment (derrabus) [inline .github/workflows/coding-standards.yml]: TODO: Change to `3.0.0` as soon as the tag has been created.

Comment (greg0ire) [reply .github/workflows/coding-standards.yml]: Done

**Review by greg0ire — DISMISSED** [review]

**Comment by greg0ire** [comment]
This needs to be updated as well I think: https://github.com/doctrine/orm/blob/b391431a0e9157b0b1760be49524d156372b3f92/phpstan-params.neon#L11

**Comment by derrabus** [comment]
Of course, done.

File: phpstan-baseline.neon [hunk 2]
```diff
@@ -510,6 +510,11 @@ parameters:
 			count: 1
 			path: lib/Doctrine/ORM/Query/ResultSetMappingBuilder.php
 
+		-
+			message: "#^Access to an undefined property Doctrine\\\\ORM\\\\Query\\\\AST\\\\Node\\:\\:\\$pathExpression\\.$#"
+			count: 1
+			path: lib/Doctrine/ORM/Query/SqlWalker.php
+
```
Comment (derrabus) [inline phpstan-baseline.neon:517]: I have no idea why this error suddenly appears when I bump the language level. But it's valid anyway.

**Review by greg0ire — APPROVED** [review]

**Pull request merged**